sc/social/like/utils.py

```diff
@@ -112,3 +117,34 @@ def get_valid_objects(brains):
             logger.warn(msg.format(b.getPath()))
             continue
         yield obj
+
+
+def get_image_fallback(site=None):
+    from sc.social.like.interfaces import ISocialLikeSettings
+    if site is None:
+        site = getSite()
+    registry = getUtility(IRegistry)
```

Comment:
I prefer to use the API here also as we are getting just one record.

Reply:
ok